packages/file-adapters/lib/local-file.js

```diff
@@ -50,6 +50,24 @@ module.exports = class LocalFileAdapter {
     );
   }
 
+  /**
+   * Takes a file data structure such as existingItem in a hook callback.
+   */
+  delete({ file }) {
+    if (file) {
+      try {
+        fs.unlinkSync(path.join(this.src, file.filename));
+      } catch (err) {
+        // -4058 is the 'no such file or directory' error.
+        if (err.errno === -4058) {
```

Comment:
We should probably use `err.code === "ENOENT"` here rather than checking the numerical value.